.ci/createBadges.py

```diff
@@ -0,0 +1,128 @@
+import json
```

Comment:
why the ci and github workflows are in this PR. is there a need to update branch or close/open pr again?

Reply:
i branched off main branch so the workflow codes are not in devmain